test/integration/server.cc

```diff
@@ -47,7 +47,7 @@ IntegrationTestServer::create(const std::string& config_path,
                               const Network::Address::IpVersion version,
                               std::function<void()> pre_worker_start_test_steps, bool deterministic,
                               Event::TestTimeSystem& time_system) {
-  IntegrationTestServerPtr server{new IntegrationTestServer(time_system, config_path)};
+  IntegrationTestServerPtr server{new IntegrationTestServerImpl(time_system, config_path)};
```

Comment:
Want to update this to use `std::make_unique` while you're making this change?

Reply:
Done.